Update Google Install and Release Notes with Core Bundle Note for 2.3.3 (#5348)
magento/devdocs: extensions/google-shopping-ads/index.md

title: Install Google Shopping ads Channel
---

{:.bs-callout .bs-callout-info}
Beginning with Magento version 2.3.3, Google Shopping ads Channel installs with the Magento update and does not require separate install. [Onboarding](https://docs.magento.com/m2/ee/user_guide/sales-channels/google-ads/onboarding-google.html) and management processes remain the same. Merchants using previous Magento versions (2.2.4+, 2.3.0, 2.3.1, and 2.3.2) can install or upgrade the Google Shopping ads Channel extension via the [Magento Marketplace](https://marketplace.magento.com/magento-google-shopping-ads.html).

> Google Shopping ads adds Google Shopping features to Magento. To review additional information, see the [Google Shopping ads Channel Marketplace page](http://marketplace.magento.com/magento-google-shopping-ads.html) and [Google Shopping ads Channel](https://docs.magento.com/m2/ee/user_guide/sales-channels/google-ads/google-ad-channel.html).

Reviewer: ads adds? Tongue twisters on DevDocs! Maybe can say:
```suggestion
The Google Shopping ads channel integrates Google Shopping features with Magento. See [Google Shopping ads Channel Marketplace](http://marketplace.magento.com/magento-google-shopping-ads.html) and [Google Shopping ads Channel](https://docs.magento.com/m2/ee/user_guide/sales-channels/google-ads/google-ad-channel.html).
```
